src/traveler_dashboard_helper.js

```diff
@@ -17,12 +20,104 @@ function renderTripsHeader(title) {
   document.getElementsByTagName("BODY")[0].appendChild(upcomingTripsSection);
 }
 
+function renderAgentWelcome() {
+  document.getElementById("login-form").style.display = "none";
+  document.getElementById("footer").style.display = "none";
+  var welcomeHeader = document.createElement("H1");
+  var headerText = document.createTextNode("Welcome, AGENT");
+  welcomeHeader.appendChild(headerText);
+  document.body.appendChild(welcomeHeader);
+}
+
+function getPendingTrips(trips) {
+  let i;
+  let pendingTripsArr = [];
```

Comment:
try to avoid adding "arr" on the names of variables.  Simply calling this "pendingTrips" implies it is an array of trips